src/crypto/sign.js

```diff
@@ -346,3 +346,42 @@ export function verifySecondSignature(transaction, publicKey) {
 
 	return res;
 }
+
+/**
+ * @method aesEncrypt
+ * @param passphrase
+ * @param password
+ *
+ * @return {string}
+ */
+
+export function aesEncrypt(passphrase, password) {
+	const iv = crypto.randomBytes(16);
+	const passHash = crypto.createHash('sha256').update(password, 'utf8').digest();
+	const cipher = crypto.createCipheriv('aes-256-cbc', passHash, iv);
+	const encrypted = cipher.update(passphrase);
+	const encryptedString = Buffer.concat([encrypted, cipher.final()]);
+	const cipherText = ''.concat(iv.toString('hex'), '$', encryptedString.toString('hex'));
```

Comment:
Since the IV is always the same length we can save a character by skipping the `$` here.

Reply:
I like the delimiter. It leaves room to change the length of the IV at a certain point and gives a clear structure